lnbits/core/static/js/wallet.js

```diff
@@ -248,8 +248,15 @@ new Vue({
       this.sendCamera.show = false;
     },
     decodeInvoice: function () {
+      if (this.send.data.bolt11.startsWith('lightning:')) {
+        this.send.data.bolt11 = this.send.data.bolt11.slice(10);
+      } else if (this.send.data.bolt11.startsWith('bitcoin:')) {
+        this.send.data.bolt11 = this.send.data.bolt11.slice(8);
```

Comment:
According to [bip121](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) `bitcoin:<ln-invoice>` isn't a valid bitcoin URI. I couldn't find a formal definition of the bip21 LN extension, but https://github.com/LN-Zap/zap-desktop/pull/3131 looks like a good example how it's handled in practice.

Reply:
Correct me if I'm wrong... `LNbits` would not send to a Bitcoin address here, right?

In that case I would limit this PR to only add `'lightning:'` - agreed?

Reply:
It wouldn't send on-chain tx, but there is a fallback scheme for bip21 URIs that looks like this: `bitcoin:bc1…?lightning=lnbc1…`. It's described in LN-Zap/zap-desktop#3131.